scripts/west_commands/sign.py

```diff
@@ -305,14 +312,15 @@ def get_cfg(command, bcfg, item):
             return None
 
     @staticmethod
-    def edt_flash_node(b):
+    def edt_flash_node(b, quiet=False):
         # Get the EDT Node corresponding to the zephyr,flash chosen DT
         # node; 'b' is the build directory as a pathlib object.
 
         # Ensure the build directory has a compiled DTS file
         # where we expect it to be.
         dts = b / 'zephyr' / 'zephyr.dts'
-        log.dbg('DTS file:', dts, level=log.VERBOSE_VERY)
+        if not quiet:
+            log.dbg('DTS file:', dts, level=log.VERBOSE_VERY)
```

Comment:
I really doubt this works. I also tried to https://pylint.pycqa.org/en/latest/user_guide/messages/warning/logging-not-lazy.html  somewhere else but it did not work; this is not the standard python logger unfortunately.

Reply:
> I really doubt this works.

This PR was merged over 2 years ago; please be more specific or file a bug :)

Reply:
Sorry for the noise, I didn't expect `log.dbg()` to work like this and notably to take a `level` argument while the other... levels (!) don't take any. It's not symmetric and intuitive.

What I should definitely have expected from you is for all that to be thoroughly documented:
https://docs.zephyrproject.org/1.14.0/guides/west/west-apis.html#module-west.log

My bad!